<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraWidget</class>
 <widget class="QWidget" name="CameraWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="TransformsGrid" native="true">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>36</y>
     <width>283</width>
     <height>210</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0,1,0,0,1,0,0,0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="PositionL">
      <property name="text">
       <string>Position:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="TripletController" name="PositionTriplet" native="true"/>
    </item>
    <item>
     <widget class="QLabel" name="RotationL">
      <property name="text">
       <string>Orientation:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="TripletController" name="RotationTriplet" native="true"/>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ScaleL">
      <property name="text">
       <string>Focus point:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="TripletController" name="ScaleTriplet" native="true"/>
    </item>
    <item>
     <widget class="QLabel" name="CameraModeL">
      <property name="text">
       <string>Camera mode:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="CameraModeGrid">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="CameraModeFree">
        <property name="text">
         <string>Free</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="CameraModeFocus">
        <property name="text">
         <string>Focus</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="ViewModeL">
      <property name="text">
       <string>View mode:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="ViewModeGrid">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="ViewModePerspective">
        <property name="text">
         <string>Perspective</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ViewModeParallel">
        <property name="text">
         <string>Parallel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="CameraL">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TripletController</class>
   <extends>QWidget</extends>
   <header location="global">tripletcontroller.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
